### DISCUSSION TITLE
方便加筆記功能嗎？

### DISCUSSION THREAD

**Discussion by Afiveqin** [SiongSng/cap-countdown#106] [提點子]
方便加筆記功能嗎？想做筆記，但不想離開這個app，怕分心

**Comment by SiongSng** [comment]
關於筆記功能我一直有在想怎麼做會更符合需求，你想要的筆記功能是可以在每個題目上手繪內容的嗎？

**Comment by Afiveqin** [comment]
抱歉這麼晚才看到，是的

菘菘 ***@***.***> 於 2023年9月17日 週日 17:22 寫道：

**Comment by SiongSng** [comment]
那應該與 #95 有一點像

**Comment by Afiveqin** [comment]
嗯嗯，差不多

菘菘 ***@***.***> 於 2023年10月5日 週四 17:16 寫道：
